### DISCUSSION TITLE
elasticseach: autocomplete for lucene query

### DISCUSSION THREAD

**Discussion by gabor** [grafana/grafana#64940] [Observability]
we could offer autocomplete in the lucene query editor.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82826. Feel free to continue the discussion around this there. Thank you!
